all-products.html

```diff
@@ -19,85 +40,96 @@ <h1>All Products</h1>
-                        <div class="image">Image will go here</div>
-                        <h2>Product name</h2>
-                        <p>Price</p>
+                        <div class="image"><a href="#"><img src="placeholder.png"></a></div>
+                            <a class="h2" href="#">Product name</a>
+                            <p>Price</p>
+
                     </div>
 
                     <div class="product-box">
-                        <div class="image">Image will go here</div>
-                        <h2>Product name</h2>
-                        <p>Price</p>
+                        <div class="image"><a href="#"><img src="placeholder.png"></a></div>
+                            <a class="h2" href="#">Product name</a>
+                            <p>Price</p>
+
                     </div>
 
                     <div class="product-box" id="first">
-                        <div class="image">Image will go here</div>
-                        <h2>Product name</h2>
-                        <p>Price</p>
+                        <div class="image"><a href="#"><img src="placeholder.png"></a></div>
+                            <a class="h2" href="#">Product name</a>
+                            <p>Price</p>
+
                     </div>
 
                     <div class="product-box">
-                        <div class="image">Image will go here</div>
-                        <h2>Product name</h2>
-                        <p>Price</p>
+                        <div class="image"><a href="#"><img src="placeholder.png"></a></div>
+                            <a class="h2" href="#">Product name</a>
+                            <p>Price</p>
+
                     </div>
 
                     <div class="product-box">
-                        <div class="image">Image will go here</div>
-                        <h2>Product name</h2>
-                        <p>Price</p>
+                        <div class="image"><a href="#"><img src="placeholder.png"></a></div>
+                            <a class="h2" href="#">Product name</a>
+                            <p>Price</p>
+
                     </div>
 
                     <div class="product-box">
-                        <div class="image">Image will go here</div>
-                        <h2>Product name</h2>
+                        <div class="image"><a href="#"><img src="placeholder.png"></a></div>
+                        <a class="h2" href="#">Product name</a>
                         <p>Price</p>
+
                     </div>
                 </div> 
 
                 <div class="nav-buttons">
-                    <!--Previous-->
-                        <a href="#">&#5176;&#5176;</a>
+                     <!--Previous-->
+                     <a href="#" class="page1">&#5176;&#5176;</a>
 
                     <!--Next-->
-                    <a href="#">&#5171;&#5171;</a>
+                    <a href="products-2.html">&#5171;&#5171;</a>

```

Comment:
In the final layout, I created a new page which is page 2 of the "All products" page and can be visited through this button